iogt_users/templates/modeladmin/iogt_users/user/index.html

```diff
@@ -0,0 +1,106 @@
+{% extends "wagtailadmin/base.html" %}
```

Comment:
Is this template copied from somewhere? Is it possible to extend instead and only change the header block to add the user count? Otherwise, it's hard to see where the customisation is, and potentially we are opening ourselves up to maintaining the whole template when we don't have to.